src/vs/workbench/services/extensions/electron-browser/localProcessExtensionHost.ts

```diff
@@ -182,18 +186,32 @@ export class LocalProcessExtensionHost implements IExtensionHost {
 					opts.execArgv = ['--inspect-port=0'];
 				}
 
-				// Enable the crash reporter depending on environment for local reporting
-				const crashesDirectory = this._environmentService.crashReporterDirectory;
-				if (crashesDirectory) {
-					const crashReporterOptions: CrashReporterStartOptions = {
+				// On linux crash reporter needs to be started on child processes explicitly
+				if (platform.isLinux) {
```

Comment:
So this seems specific to Linux, even with Electron 9 we need to start the crash reporter?

Reply:
Only on linux for child node processes the crash reporter has to be started explicitly due to implementation reasons, I have clarified the comment section in the code.